source/_integrations/supla.markdown

```diff
@@ -44,4 +48,12 @@ access_token:
     `http[s]://your.server.org/integrations/tokens` (please add at least Channel's Read and Action Execution permissions).
   required: true
   type: string
+scan_interval:
```

Comment:
I cannot see the change in the parent PR that adds this parameter to the YAML?

Reply:
Due to ADR10 I had to remove this change, I'm removing this pull request.